charts/orchestrator/templates/network-policies.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.sonataFlowOperator.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-sonata-rhdh-to-workflows
+  namespace: {{- include "get-workflow-namespace" . }}
+spec:
+  podSelector: {}
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: {{ .Values.rhdhOperator.subscription.namespace }}
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: {{ .Values.orchestrator.namespace }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-rhdh-and-workflows-to-sonata
+  namespace: {{ .Values.orchestrator.namespace }}
```

Comment:
This CR needs to be created in the namespace that runs sonataflow services (Data Index).
Based on the chart, is should rely on `{{- include "get-workflow-namespace" . }}` instead of `{{ .Values.orchestrator.namespace }}`

Reply:
orchestrator namespace is where sonata is installed and I see it has a default value of sonataflow-infra

Reply:
ok I see what you mean by looking at the clusterplatform. this is bit confusing
in that case they should always be the same namespace unless cluster platform is set.